projects/observability/src/shared/components/explore-query-editor/explore-query-editor.component.ts

```diff
@@ -56,7 +56,7 @@ import {
           >
           </ht-explore-query-limit-editor>
         </div>
-        <div class="filters-row" *htIfFeature="'${ApplicationFeature.ExplorerOrderBySupport}' | htFeature">
```

Comment:
this needs to be restored right?

Reply:
yes, ignore it for now. I made a mistake when I push it. I have pending one change yet.

Reply:
rolled back